website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/setup-azure.md

```diff
@@ -101,16 +100,186 @@ Once you connect your Azure AD app and Azure DevOps, you need to provide dbt Clo
+Because Azure DevOps forces all authentication to be linked to a user's permissions, we recommend you create a "service user" in Azure DevOps whose permissions which will be used to power headless actions in dbt Cloud such as deployment runs and CI. A service user is a pseudo user set up in the same way an admin would set up a real user, but it's given permissions specifically scoped for service to service interactions. You should avoid linking authentication to a real Azure DevOps user because if this person leaves your organization, dbt Cloud will lose privileges to the dbt Azure DevOps repositories, causing production runs to fail.
+
+### More on Service Users
+
+A service user account must have the following permissions for all Azure DevOps projects/repos you want accessible in dbt Cloud:
 
-A "service user" account is a pseudo user account in Azure DevOps with read access to all dbt repos for the whole dbt Cloud account. This account enables you to scope permissions appropriately and preserve access. 
+ - **Project Reader**
+ - **ViewSubscriptions**
+ - **EditSubscriptions**
+ - **DeleteSubscriptions** * 
+ - **PullRequestContribute**
+ - **GenericContribute**
 
-If you don't want to create a separate Azure DevOps user, you can link scheduled runs to an admin profile who has appropriate read access to the dbt repositories, but we don't recommended approach, as you could over permission the service user or risk losing read access should the admin leave your organization. dbt Cloud will refresh the service user's OAuth access token regularly behind the scenes.
+\* Note: **DeleteSubscriptions** permission might be included in **EditSubscriptions** depending on your version of Azure.
 
-:::info  Azure DevOps admin must grant read access to the service user
-This service user's permissions will also power which repos a team can select from during dbt project set up, so an Azure DevOps admin must grant read access to the service user before setting up a project in dbt Cloud.
+Some of these permissions are only accessible via the Azure DevOps API, for which documentation can be found [here](https://docs.microsoft.com/en-us/azure/devops/organizations/security/namespace-reference?view=azure-devops). We’ve also detailed more information on Azure DevOps API usage below to help accelerate the set up. Alternatively, you can use the Azure DevOps UI to enable permissions, but you cannot get the least permissioned set.
+
+:::info  The service user must have these permissions before dbt Cloud Project set up
+This service user's permissions will also power which repos a team can select from during dbt project set up, so an Azure DevOps admin must grant at minimum Project Reader access to the service user before setting up a project in dbt Cloud.
 :::
 
+<details>
```

Comment:
@schottj I think this permission info would be a lot easier to read as a table. Let me know if you're OK with that and I will update!

Reply:
I tried this at first, but I could get the table entries to show code snippets. If you figure it out, would love help re-arranging!

Reply:
Just to follow up: I looked into making a table here and we'd need to use an HTML table. This is doable but will take longer. So let's ship this and follow up later when we have time.